src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,18 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <p>
+        {' Posted by  '}
+        <UserInfo user={post.user} />
```

Comment:
```suggestion
        { post.user && <UserInfo user={post.user} />}
```
technically post.user can be undefined. suggest to render UserInfo component conditionally